mitodl/open-discussions
# fix display of report count, fix too large font size

#### Pre-Flight checklist



#### What are the relevant tickets?

none

#### What's this PR do?

Fix a little styling regression, so this doesn't happen:

![stylegregress](https://user-images.githubusercontent.com/6207644/45713195-5c9d5400-bb5c-11e8-993c-5d2068272feb.png)


#### How should this be manually tested?

make reports on a post, and make sure the report count widget looks normal on the compact and expanded post display components.
